src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -516,13 +567,17 @@ void Frustum::pup(PUP::er& p) {
     p | half_opening_angle_;
     p | one_over_tan_half_opening_angle_;
   }
+  if (version >= 2) {
+    p | inner_radius_;
+    p | zeta_distribution_;
```

Comment:
In `version < 2` you have to unpack a bool where the `with_projective_map_` data was and set `zeta_distribution_` corresponding to that bool.